src/main/content/_assets/css/404page.css

```diff
@@ -0,0 +1,364 @@
+    top: 43.3%;
+    right: 9.7%;
+}
+
+#left_planet {
+    top: 42%;
+    left: 13.8%;
+}
+
+#smstar_br {
+    bottom: 1%;
+    right: 3.2%;
+}
+
+#smstar_bl {
+    bottom: 6.1%;
+    left: 3.3%;
+}
+
+#smstar_c {
+    top: 40%;
+    left: 47%;;
+}
+
+#medstar_l {
+    top: 24%;
+    left: 14.6%;
+    -webkit-animation: blink 4s infinite;
+            animation: blink 4s infinite;
+}
+
+#medstar_r {
+    top: 8.6%;
+    right: 32%;
+    -webkit-animation: blink 4s infinite;
+            animation: blink 4s infinite;
+    -webkit-animation-delay: -2s;
+            animation-delay: -2s;
+}
+
+#bigstar {
+    bottom: 30.4%;
+    right: 7.6%;
+    -webkit-animation: blink 4s infinite;
+            animation: blink 4s infinite;
+    -webkit-animation-delay: -3s;
+            animation-delay: -3s;
+}
+
+#star_cluster1 {
+    top: 4.9%;
+    left: 0.8%;
+    -webkit-transform: rotateY(180deg);
+            transform: rotateY(180deg);
+}
+
+#star_cluster2 {
+    top: 16.6%;
+    right: 2.4%;
+    -webkit-transform: rotateX(180deg);
+            transform: rotateX(180deg);
+}
+
+#star_cluster3 {
+    top: 61.3%;
+    left: 9.7%;
+    -webkit-animation: blink 4s infinite;
+            animation: blink 4s infinite;
+    -webkit-animation-delay: -1s;
+            animation-delay: -1s;
+}
+
+/* Blinking stars animation*/
+@-webkit-keyframes blink {
+    0% {
+        opacity: 0;
+    }
+    10%, 40% {
+        opacity: 1;
+    }
+    60%, 100% {
+        opacity: 0;
+    }
+}
+@keyframes blink {
+    0% {
+        opacity: 0;
+    }
+    10%, 40% {
+        opacity: 1;
+    }
+    60%, 100% {
+        opacity: 0;
+    }
+}
+
+/* Blinking smoke animation */
+@-webkit-keyframes smoke_blink {
+    0% {
+        opacity: 0;
+    }
+    10% {
+        opacity: 1;
+    }
+    60%, 100% {
+        opacity: 0;
+    }
+}
+@keyframes smoke_blink {
+    0% {
+        opacity: 0;
+    }
+    10% {
+        opacity: 1;
+    }
+    60%, 100% {
+        opacity: 0;
+    }
+}
+
+/* Blinking ship lights animation */
+@-webkit-keyframes shiplight_blink {
+    0%, 20%, 60%, 80%, 100% {
+        opacity: 0;
+    }
+    10%, 30%, 45%, 70% {
+        opacity: 1;
+    }
+}
+@keyframes shiplight_blink {
+    0%, 20%, 60%, 80%, 100% {
+        opacity: 0;
+    }
+    10%, 30%, 45%, 70% {
+        opacity: 1;
+    }
+}
+
+/* Override navbar colors */
+#navbar_responsive_button {
+    background: none;
+}
+
+@media (max-width: 767px) {
+    .navbar {
+        background-color: #1f203e;
+    }
+}
+
+
+/* Mobile and tablet styling */
+@media (max-width: 576px) {
+    #page_container {
+        padding-top: 113px;
+    }
+
+    #error_message {
+        margin-bottom: 114px;
+        font-size: 16px;
+        width: 230px;
+        max-width: 100%;
+    }
+
+    #crashed_ship_container {
+        width: 429px;
+        height: 194px;
+    }
+
+    #crashed_ship {
+        width: 429px;
+        height: 194px;
+    }
+
+    #shiplight_top, #shiplight_mid, #shiplight_bot {
```

Comment:
This may be personal preference, but for organizational purpose, I would use a class like `.shiplight` on these elements, and for common styling across them.